### PR TITLE
https://issues.redhat.com/browse/ACM-6001 Add RHCOS image limits

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

**Comment by cadawson** [comment]
Hi @elsapassaro and @jelkosz . Can you review this quick update about RHCOS requirements? Thanks!

**Comment by jelkosz** [comment]
@filanov PTAL

**Comment by filanov** [comment]
LGTM

**Comment by jelkosz** [comment]
adding @filanov for this one

File: clusters/cluster_lifecycle/create_infra_env.adoc [hunk 1]
```diff
@@ -83,9 +83,11 @@ Registries in the list of `unqualified-search-registries` are automatically adde
 
 . Create the `AgentServiceConfig` custom resource by completing the following steps:
 
+*Important:* If you are using the late binding feature, the {ocp-short} release images that you use when creating your clusters must be version 4.13 or later when the `spec.osImages` releases that are specified in the the `AgentServiceConfig` custom resource are version 4.13 or later. The Red Hat Enterprise Linux CoreOS images for version 4.13 and later are not compatible with images earlier than version 4.13.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_infra_env.adoc]:
```suggestion
*Important:* If you are using the late binding feature, the {ocp-short} release images that you use when creating your clusters must be version 4.13 or later when the `spec.osImages` releases that are specified in the `AgentServiceConfig` custom resource are version 4.13 or later. The Red Hat Enterprise Linux CoreOS images for version 4.13 and later are not compatible with images earlier than version 4.13.   
```

Comment (dockerymick) [reply clusters/cluster_lifecycle/create_infra_env.adoc]: Alternative

```suggestion
*Important:* If you are using the late binding feature and the `spec.osImages` releases in the `AgentServiceConfig` custom resource are version 4.13 or later, the {ocp-short} release images that you use when creating your clusters must be version 4.13 or later. The Red Hat Enterprise Linux CoreOS images for version 4.13 and later are not compatible with images earlier than version 4.13.   
```

**Review by dockerymick — CHANGES_REQUESTED** [review]
Made a couple of suggestions

**Comment by cadawson** [comment]
Like the suggestion. Thanks @dockerymick!

**Review by dockerymick — APPROVED** [review]

**Pull request merged**